backend/python/Dockerfile

```diff
@@ -6,6 +6,7 @@ COPY requirements.txt ./
 RUN pip install -r requirements.txt
 
 COPY . ./app
+VOLUME file_uploads:/var/lib/file_uploads
```

Comment:
When I tested out this branch, the `file_uploads` directory did not exist. I was getting the previous issue we had: `No such file or directory: '/var/lib/file_uploads/setup.sh`. After I mkdir'd the directory in the docker container, I could start properly uploading files. 

Did you experience something similar? Maybe we need to do something like `RUN sh -c ‘touch /var/lib/file_uploads’ `